Assets/Mirror/Runtime/NetworkConnection.cs

```diff
@@ -139,27 +143,23 @@ internal virtual void Send(ArraySegment<byte> segment, int channelId = Channels.
         {
             //Debug.Log("ConnectionSend " + this + " bytes:" + BitConverter.ToString(segment.Array, segment.Offset, segment.Count));
 
-            // validate packet size first.
-            if (ValidatePacketSize(segment, channelId))
-            {
-                // batching enabled?
-                if (batching)
-                {
-                    // add to batch no matter what.
-                    // batching will try to fit as many as possible into MTU.
-                    // but we still allow > MTU, e.g. kcp max packet size 144kb.
-                    // those are simply sent as single batches.
-                    //
-                    // IMPORTANT: do NOT send > batch sized messages directly:
-                    // - data race: large messages would be sent directly. small
-                    //   messages would be sent in the batch at the end of frame
-                    // - timestamps: if batching assumes a timestamp, then large
-                    //   messages need that too.
-                    GetBatchForChannelId(channelId).AddMessage(segment);
-                }
-                // otherwise send directly to minimize latency
-                else SendToTransport(segment, channelId);
-            }
+            // add to batch no matter what.
+            // batching will try to fit as many as possible into MTU.
+            // but we still allow > MTU, e.g. kcp max packet size 144kb.
+            // those are simply sent as single batches.
+            //
+            // IMPORTANT: do NOT send > batch sized messages directly:
```

Comment:
Is this still a problem? I thought it was fixed

Reply:
yes it's fixed.
it's just a reminder to not do that again, and why :)